src/v2i-hub/JSONMessageLoggerPlugin/src/JSONMessageLoggerPlugin.cpp

```diff
@@ -72,35 +72,29 @@ namespace JSONMessageLoggerPlugin {
     {
         tmx::utils::TmxMessageManager::OnMessageReceived(msg);
         tmx::routeable_message routeMsg(msg);
+        PLOG(tmx::utils::logDEBUG1) << "Routable Message " << routeMsg.to_string();
         // Cast routeable message as J2735 Message
         if (tmx::utils::PluginClient::IsJ2735Message(routeMsg)) {
             try {
-                // Convert routeable message to J2735 encoded message
-                tmx::messages::TmxJ2735EncodedMessage<tmx::messages::MessageFrameMessage> rMsg = 
-                    routeMsg.get_payload<tmx::messages::TmxJ2735EncodedMessage<tmx::messages::MessageFrameMessage>>();
-                // Decode Encode J2735 Message
-                auto j2735Data = rMsg.decode_j2735_message().get_j2735_data();
-                // Convert J2735 data to TmxJ2735Message for JSON serialization
-                auto j2735Message = tmx::messages::TmxJ2735Message<MessageFrame_t, tmx::JSON>(j2735Data);
-                // Serial J2735 message to JSON
-                std::string json_payload_str = j2735Message.to_string();
-                ASN_STRUCT_FREE(asn_DEF_MessageFrame, j2735Data.get());
+                if (routeMsg.get_flags() & IvpMsgFlags_RouteDSRC) {
+                    PLOG(tmx::utils::logDEBUG1) << "Logging TX J2735 Message";
+                    logRouteableMessage(routeMsg, txLogger);
 
-                // Free the J2735 data structure
-                PLOG(tmx::utils::logDEBUG1) << json_payload_str;
-           
-                if ( routeMsg.get_flags() & IvpMsgFlags_RouteDSRC ) {
-                    BOOST_LOG_SEV(txLogger, boost::log::trivial::info) << json_payload_str;
                 }
                 else {
-                    BOOST_LOG_SEV(rxLogger, boost::log::trivial::info) << json_payload_str;
+                    PLOG(tmx::utils::logDEBUG1) << "Logging RX J2735 Message";
+                    logRouteableMessage(routeMsg, rxLogger);
                 }
             }
             catch (const boost::exception &e) {
-                PLOG(tmx::utils::logERROR) << "Boost exception while logging message: " << boost::diagnostic_information(e);
-            }
-            catch (const std::exception &e) {
-                PLOG(tmx::utils::logERROR) << "Exception while logging message: " << e.what();
+                std::string errorMessage = "Boost exception while logging message: " + boost::diagnostic_information(e);
+                FILE_LOG(tmx::utils::logERROR) << errorMessage;
```

Comment:
FILE_LOG vs PLOG?  What is the difference in terms of when to use file log when when to use plugin log?